src/styles/buy.css

```diff
@@ -0,0 +1,25 @@
+.card__buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  margin-top: 16px;
+  border-radius: 5px;
+  width: 166px;
+  height: 40px;
+
+  border: 1px solid #00acdc;
+  background: #00acdc;
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 700;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #fff;
+}
+
+.card__buy:hover {
+  border: 1px solid #00acdc;
```

Comment:
It looks like we shouldn't set the full border property for the button hover state. The border should be set transparent to the button's normal state and only its color should change on hover. Checklist p8.